lib/Doctrine/DBAL/Driver/PDOStatement.php

```diff
@@ -213,7 +216,13 @@ public function fetchColumn($columnIndex = 0)
     private function convertParamType(int $type) : int
     {
         if (! isset(self::PARAM_TYPE_MAP[$type])) {
-            throw new \InvalidArgumentException('Invalid parameter type: ' . $type);
+            // TODO: next major: throw an exception
+            @trigger_error(sprintf(
+                'Using a PDO parameter type (%d given) is deprecated and will cause an error in Doctrine 3.0',
```

Comment:
Can we have a test for this one too? 🙏

Reply:
PostgeSQL doesn't like `PDO::PARAM_INPUT_OUTPUT` (see https://travis-ci.org/doctrine/dbal/jobs/363396885).